### DISCUSSION TITLE
Can I resizing the image and make itinto a circle

### DISCUSSION THREAD

**Discussion by LAPIII** [ImageMagick/ImageMagick#4979] [Help]
I want to resize an image to 100 x 100 pixels and then make it into a circle. I haven't used this software in a long time so please tell me exactly what to type. Thank you.

**Comment by snibgo** [comment]
>  ... so please tell me exactly what to type.

That depends on what version of IM you use, on what platform.

**Reply by LAPIII** [reply]
I'm sorry I didn't give a better description I'm on Windows.

**Comment by snibgo** [comment]
What version of IM do you use?

**Reply by LAPIII** [reply]
Sorry, I had updated to ImageMagick 7.1.0 Q16

**Comment by snibgo** [comment]
How about this:
```
magick toes.png -resize "100x100^!" ( -size 100x100 xc:Black -fill White -draw "circle 50,50,99,50" ) -alpha off -compose CopyAlpha -composite out.png
```
![out](https://user-images.githubusercontent.com/33812368/159592784-b4fde17d-d5f9-4a4c-918b-7cfe0cb8ea78.png)

**Reply by LAPIII** [reply]
Do images to convert have to be stored in a particular location? Is there documentation for this app?

**Comment by snibgo** [comment]
> Do images to convert have to be stored in a particular location?

No. If they are not in your current directory, give the path to the files.

> Is there documentation for this app?

Yes. For example:
http://imagemagick.org/Usage/
http://www.imagemagick.org/script/command-line-options.php
http://imagemagick.org/Usage/basics/
http://imagemagick.org/Usage/windows/
